host/xtest/regression_1000.c

```diff
@@ -1010,6 +1072,119 @@ static void xtest_tee_test_1012(ADBG_Case_t *c)
 
 		TEEC_CloseSession(&session1);
 	}
+	Do_ADBG_EndSubCase(c, "Single Instance Multi Session");
+
+	Do_ADBG_BeginSubCase(c, "Multiple Instances Single Session");
+	{
+		uuid = miss_test_ta_uuid;
+
+		(void)ADBG_EXPECT_TEEC_SUCCESS(c, open_session(&cs[0], uuid));
+		(void)ADBG_EXPECT_TEEC_SUCCESS(c, open_session(&cs[1], uuid));
+
+		(void)ADBG_EXPECT_TEEC_RESULT(c, TEEC_SUCCESS,
+					do_get_counter(&cs[0], &counter));
+		(void)ADBG_EXPECT(c, 0, counter);
```

Comment:
If we don't care about the value, you can remove this test, just check the command succeeded.